Assignment03.java

```diff
@@ -0,0 +1,236 @@
+
+            switch (choice) {
+
+                case 1:
+                    System.out.println("Enter how many employees you want: ");
+                    int n = integer_input.nextInt();
+                    for (int i = 0; i < n; i++) {
+                        System.out.print("Enter Employee Name: ");
+                        String name = string_input.nextLine();
+
+                        System.out.print("Enter Employee Age: ");
+                        int age = integer_input.nextInt();
+
+                        System.out.print("Enter Employee Date of birth: ");
+                        String date_of_birth = string_input.nextLine();
+
+                        System.out.print("Enter Employee email address: ");
+                        String email_address = string_input.nextLine();
+
+                        array_list.add(new Employee(name, age, date_of_birth, email_address));
+                    }
+                    object_output_stream = new ObjectOutputStream(new FileOutputStream(file));
+                    object_output_stream.writeObject(array_list);
+                    object_output_stream.close();
+
+                    break;
+
+                case 2:
+                    if (file.isFile()) {
+                        object_input_stream = new ObjectInputStream(new FileInputStream(file));
+                        array_list = (ArrayList<Employee>) object_input_stream.readObject();
+                        object_input_stream.close();
+
+                        System.out.println("----------------------------------------");
+                        list_iterator = array_list.listIterator();
+                        while (list_iterator.hasNext())
+                            System.out.println(list_iterator.next());
+                        System.out.println("----------------------------------------");
+                    } else {
+                        System.out.println("File not Exist...!!");
+                    }
+                    break;
+
+                case 3:
+                    if (file.isFile()) {
+                        object_input_stream = new ObjectInputStream(new FileInputStream(file));
+                        array_list = (ArrayList<Employee>) object_input_stream.readObject();
+                        object_input_stream.close();
+
+                        boolean found = false;
+                        System.out.print("Enter name to delete: ");
+                        String name = string_input.nextLine();
+                        System.out.println("----------------------------------------");
+                        list_iterator = array_list.listIterator();
+                        while (list_iterator.hasNext()) {
+                            Employee employee = (Employee) list_iterator.next();
+                            if (employee.name.equals(name)) {
+                                list_iterator.remove();
+                                found = true;
```

Comment:
If record is found then you can break this loop otherwise the whole loop will be executed.